cmd/podman/containers/ps.go

```diff
@@ -300,6 +295,7 @@ func createPsOut() ([]map[string]string, string) {
 		"PIDNS":        "pidns",
 		"Pod":          "pod id",
 		"PodName":      "podname", // undo camelcase space break
+		"RestartCount": "restarts",
```

Comment:
I prefer this to be part of another commit.  Adding restarts to pods shouldn't require changes to `container ps`.

Reply:
Adding restarts to podman ps is part of the design doc goals. It also helps with testing the pod restart policy feature.

I separated out the commits for podman ps, podman pod ps, and podman pod create --restart.

Reply:
> Adding restarts to podman ps is part of the design doc goals

Apologies, I did not see that.

> It also helps with testing the pod restart policy feature.

The `inspect` commands already support showing the restart counts (which is used in other tests), so I don't think it helps with testing.

I don't feel extremely strongly about (not) adding the flag but if the motivation was to simplify testing, then we can avoid the extra code by just using `inspect`.

Reply:
I think it is still ideal for users to be able to get this info from ps, so I have left the option available for --format. I have removed the `--restart` flag for both `podman ps` and `podman pod ps`.